src/chapters/sql-part1/exercises.rst

```diff
@@ -0,0 +1,319 @@
+.. _sql-part1-exercises:
+
+Exercises: SQL, Part 1
+======================
+
+In order to practice SQL commands, you must first set up a new database (model)
+on your computer. In the MySQLWorkbench program, each new model is called a
+*schema*.
+
+#. Open MySQLWorkbench and open a connection. You may be prompted once or twice
+   for passwords, depending on how you set up your root.
+
+   .. figure:: ./figures/mswHomeScreen.png
+      :alt: MySQLWorkbench home screen.
+      :scale: 60%
+
+#. Click on the *Create New Schema* button and give the database a distinctive
+   name.
+
+   .. figure:: ./figures/createNewSchemaButton.png
+      :alt: Create new schema button.
+
+#. Click *Apply*, and accept all of the default options when prompted. You
+   should see the new database when you click the *Schemas* tab.
+
+   .. figure:: ./figures/practiceSchema.png
+      :alt: New practice database in the file tree.
+      :scale: 80%
+
+#. Your new database is *almost* ready to use. The last step is to set the
+   model as the default option for SQL commands. Do this by double-clicking its
+   name in the file tree.
+
+Good. You are set up for this lesson's practice. Complete the following
+exercises in the *Query* tab of MySQLWorkbench.
+
+.. admonition:: Tip
+
+   If you have questions about how to use MySQLWorkbench, do not hesitate to
+   reference `the documentation <https://dev.mysql.com/doc/workbench/en/wb-home.html>`__.
+
+Table Setup
+-----------
+
+For these exercises, we will use a ``seeds`` table to store information for our
+garden hopes. The table will have 4 columns:
+
+#. ``seed_id``: This is a unique number assigned to each row in the ``seeds``
+   table.
+#. ``crop``: Identifies what the seeds grow (e.g. yellow bell peppers).
+#. ``encourages``: Identifies the benefits of planting the crop (e.g. nitrogen
+   fixation, attracts bees, etc.)
+#. ``use_by``: The year the seeds expire.
+
+The the ``crop`` and ``encourages`` columns will hold string data types,
```

Comment:
Double "the"

Reply:
In the chapter, we talk MySQL not using strings and instead focus on ``varchar``

Reply:
Done.